### PR TITLE
Update Default Header text color

### PR DESCRIPTION
**Pull request by ArshdeepSingh3-eaton**
Fixes # .
Updated the Default text color for header component

#### Changes proposed in this Pull Request:
- 
- 
- 

#### Screenshots / Screen Recording (if applicable)
-
Screenshot 2023-12-12 at 2 18 15 PM

#### To Test:
- 

#### Any specific feedback you are looking for?
-

### CODE REVIEW

**Review by surajeaton — COMMENTED** [review]
Please update the changelog.

File: components/src/core/Header/Header.tsx [hunk 1]
```diff
@@ -503,7 +503,7 @@ export const Header: React.FC<HeaderProps> = (props) => {
             // @change color once have a correct color
             return theme.colors.onSurface;
         }
-        return fontColor || 'white';
+        return fontColor || theme.colors.primary;
```
Comment (huayunh) [inline components/src/core/Header/Header.tsx:506]: should be "on primary container"

**Review by huayunh — COMMENTED** [review]
You shouldn't need a changelog item. See [this comment](https://github.com/etn-ccis/blui-react-native-component-library/pull/465#pullrequestreview-1777782649).

**Comment by huayunh** [comment]
Can you also change the default icon colors as part of this PR? I know it was not part of the original issue description, but the top left icon should be "on-surface", and the action icons on the right should be "on-surface-variant". Sorry for not highlighting that earlier.

**Comment by ArshdeepSingh3-eaton** [comment]
i have updated the color but as of now the code is configured in a manner where it takes as fontColor as a prop and that specific color will be used for title, subtitle, info, and icons in the header

**Review by huayunh — APPROVED** [review]
We can call this done to avoid scope creeping, but can you log an item in the backlog to address this? @ArshdeepSingh3-eaton

**Comment by joebochill** [comment]
If you want it to be part of this version, we should fix it now. We're planning to publish at the end of the sprint, so anything on the backlog is not going to make it.

**Comment by huayunh** [comment]
If it takes less than 2 hours to fix, please go ahead and address it @ArshdeepSingh3-eaton

**Pull request merged**